.github/workflows/publish-major-minor.yml

```diff
@@ -0,0 +1,33 @@
+name: "Publish Major-Minor-Tags"
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  push-tags:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install packaging
+        run: pip install -U packaging
+      - name: Get versions
+        id: get_versions
+        run: python .github/workflows/publish-major-minor.py ${{ github.ref }}
```

Comment:
The preferable invocation style is using `python -m`. This will require renaming the module not to have dashes. Also, I'm not sure if it should be under the workflows dir but it's probably fine for now unless you want to move it back in. Trust me, f-strings do work in run-steps, I have a few repos relying on this feature :)

Reply:
Moved it in the workflow, don't know why it didn't work before 😕

Reply:
Probably you started testing without adding `actions/setup-python@v2` step and it used whatever default was set...